docs/user-guides/KOBman-usage-info.md

```diff
@@ -6,14 +6,26 @@
 
 	-  `install` - Install KOB environment on the host with X.Y.Z version number format.
 	-  `--environment/-env` - Parameter to indicate the environment that is to be installed.
-	-  `[env_name]` - Name of the environment to be installed.
+	-  `[env_name]` - Name of the [environment](https://github.com/hyperledgerkochi/KOBman/blob/master/dist/environments) to be installed.
+	-  `{version}` - The environment's [version](https://github.com/hyperledgerkochi/KOBman/blob/master/dist/list.txt) that has to be installed
+	-   `KOBMAN_USER_NAMESPACE` - need to specify your github namespace
 	
 - **Examples** :
```

Comment:
@EtricKombat  Please use this example as a note to the user saying that instead of saying it as example, "If the env is being installed for the first time, you will be prompted to enter your GitHub id." Give a snap of the message
 
After the note, give the example and the output